frontend/conf/routes

```diff
@@ -54,6 +54,7 @@ POST           /subscription/update-card          controllers.Subscription.updat
 OPTIONS        /subscription/update-card          controllers.Subscription.updateCardPreflight
 
 # Events
+GET            /whatson                           controllers.Event.whatsOn
```

Comment:
This has been my eternal question. `/whats-on` is probably clearer too, unless we mean what, son? :ear: